### DISCUSSION TITLE
All sites seen as infected

### DISCUSSION THREAD

**Discussion by rossdav70** [plausible/analytics#3679] [Anything Else]
One of my clients got told by a customer that their site is being flagged as infected on BitDefender. Having installed it myself and digging deeper I can see that ALL my sites say they are infected and for some it even blocks access. The culprit? It appears to be the Plausible Analytics plugin? I am self hosting.

Anyone else seen this?   "We blocked this dangerous page for your protection: https://xxxxxxxxxxxxx/api/event"

**Comment by ruslandoga** [comment]
👋 @rossdav70 

The solution ~~is usually~~ might be to report the false positive: https://www.bitdefender.com/consumer/support/answer/29358/

**Reply by rossdav70** [reply]
Are you answering this as someone who has experienced the problem and fixed it ... or just guessing?

**Reply by ruslandoga** [reply]
I just remembered another discussion like this :)

---

But you are right, my use of "usually" was unnecessary. Just typing without thinking :)

**Reply by ruslandoga** [reply]
This one: https://github.com/plausible/analytics/discussions/3539

**Reply by rossdav70** [reply]
Thank you! So Ive registered a false positive with bitdefender, looks like I may need to test and do the same for a whole range of AV
